src/key_bind.rs

```diff
@@ -22,7 +22,7 @@ pub fn key_binds() -> HashMap<KeyBind, Action> {
 
     // Standard key bindings
     bind!([Ctrl, Shift], Key::Character("A".into()), SelectAll);
-    bind!([Ctrl, Shift], Key::Character("C".into()), Copy);
```

Comment:
Please leave in Ctrl+Shift+C. You can do another binding for Ctrl+C like CopyOrCtrlC to make it clear.

Reply:
Ideally, would the Copy function just have two shortcuts, Ctrl+C and Ctrl+Shift+C?

Reply:
Two shortcuts, with different actions. Ctrl+Shift+C is Copy, Ctrl+C is CopyOrCtrlC. This way, Ctrl+Shift+C will always copy and never send a signal.

Reply:
I named it `CopyOrSigint` for less confusion.